go-controller/hack/lint.sh

```
@@ -1,14 +1,18 @@
 #!/usr/bin/env bash
-
 VERSION=v1.60.3
+extra_flags=""
 if [ "$#" -ne 1 ]; then
+  if [ "$#" -eq 2 ] && [ "$2" == "fix" ]; then
+    extra_flags="--fix"
+  else
     echo "Expected command line argument - container runtime (docker/podman) got $# arguments: $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)